code/game/atoms.dm

```diff
@@ -276,6 +279,7 @@ its easier to just keep the beam vertical.
 		icon_state = new_icon_state
 
 /atom/proc/update_icon()
+	SHOULD_CALL_PARENT(TRUE)
```

Comment:
Nothing else overrides or calls `update_icon()` though. It's even tested by Travis that nothing overrides it.

Reply:
I can confirm that this is not true

Reply:
I stand corrected. Thanks relative pathing.